optuna_dashboard/artifact/_backend.py

```diff
@@ -163,6 +196,25 @@ def delete_artifact(study_id: int, trial_id: int, artifact_id: str) -> dict[str,
         response.status = 204
         return {}
 
+    @app.delete("/api/artifacts/<study_id:int>/<artifact_id:re:[0-9a-fA-F-]+>")
+    @json_api_view
+    def delete_study_artifact(study_id: int, artifact_id: str) -> dict[str, Any]:
```

Comment:
Is it possible to use `artifact_store.remove` in this function?

Reply:
Actually, it has already been used in this function.

Reply:
Ops. Sorry for the confusion.